packages/shared/src/utils/config/config.ts

```diff
@@ -71,32 +71,32 @@ export const getAppId = () => {
 };
 
 export const getSocketURL = () => {
-    const local_storage_server_url = window.localStorage.getItem('config.server_url');
+    const local_storage_server_url: string | null = window.localStorage.getItem('config.server_url');
     if (local_storage_server_url) return local_storage_server_url;
 
     let active_loginid_from_url;
-    const search = window.location.search;
+    const search: string = window.location.search;
     if (search) {
-        const params = new URLSearchParams(document.location.search.substring(1));
+        const params: URLSearchParams = new URLSearchParams(document.location.search.substring(1));
         active_loginid_from_url = params.get('acct1');
     }
 
-    const loginid = window.localStorage.getItem('active_loginid') || active_loginid_from_url;
-    const is_real = loginid && !/^VRT/.test(loginid);
+    const loginid: string | null | undefined = window.localStorage.getItem('active_loginid') || active_loginid_from_url;
+    const is_real: boolean | '' | null | undefined = loginid && !/^VRT/.test(loginid);
 
-    const server = is_real ? 'green' : 'blue';
+    const server: 'green' | 'blue' = is_real ? 'green' : 'blue';
     const server_url = `${server}.binaryws.com`;
 
     return server_url;
 };
 
 export const checkAndSetEndpointFromUrl = () => {
```

Comment:
@niloo-fs All declared types can be removed from this `checkAndSetEndpointFromUrl()` function because they are inferred successfully, so it's unnecessary to declare them. Please revert line 95, line 98, line 99, line 114 & line 115 🙂